lib/CL/pocl_llvm_api.cc

```diff
@@ -1404,7 +1404,9 @@ static PassManager& kernel_compiler_passes
   }
   // Add the work group launcher functions and privatize the pseudo variable
   // (local id) accesses.
-  passes.push_back("workgroup");
+  // TODO: get the workgroup pass working with CUDA?
```

Comment:
I think the workgroup pass is not needed with CUDA. It would be best not to have CUDA branch here in the code, but have something similar to 'SPMD' property above, but I think the best of all would be to move the pass creation to the device completely so all targets could customize their kernel code generation as they see best. If you consider one of these two much effort at this point, I think this is fine for now.